packages/locale/lang/hi.ts

```diff
@@ -0,0 +1,140 @@
+export default {
+  name: 'hi',
+  el: {
+    breadcrumb: {
+      label: 'रोटी का टुकड़ा',
+    },
+    colorpicker: {
+      confirm: 'पुष्टि करें',
+      clear: 'साफ़',
```

Comment:
Could you also translate the missing items according to the [English locale file](https://github.com/element-plus/element-plus/blob/dev/packages/locale/lang/en.ts)?

Reply:
of course